tests/backends/tests.py

```diff
@@ -745,6 +745,19 @@ def test_cursor_contextmanager_closing(self):
             self.assertIsInstance(cursor, CursorWrapper)
         self.assertTrue(cursor.closed)
 
+    @skipUnlessDBFeature('test_db_allows_multiple_connections')
+    def test_is_usable_before_after_connection(self):
+        """
+        Test that is_usable() doesn't crash when the connection is not open yet.
+        Test that is_usable() doesn't crash when the connection is already closed.
+        """
+        # without a connection, the database is unusable
+        self.assertFalse(connection.is_usable())
+        with connection.cursor():
```

Comment:
Other test use the context manager to open a connection cf.
https://github.com/boblefrag/django/blob/5a1f69b637d3944803cb5fd43ed06a504720239c/tests/backends/tests.py#L770

I try to stick to it for consistency